components/details_feedback/child_components/checkbox/details_feedback_widget_checkbox.react.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Checkbox extends React.Component {
+  render() {
+    const { id, labelText, onClick, autoFocus } = this.props;
+    return (
+      <div className="checkbox">
+        <input
+          type="checkbox"
+          id={id}
+          value={id}
+          onClick={onClick}
+          autoFocus={autoFocus}
+        />
+        <label for={id}>{labelText}</label>
+      </div>
+    );
+  }
+  static propTypes = {
```

Comment:
Should `autofocus` have a default prop type too?

Reply:
fixed